config/locales/en.yml

```diff
@@ -963,36 +963,36 @@ en:
     automated_workflow_executions:
       actions:
         delete_button: Delete
-        delete_confirm: Are you sure that you want to delete this automated workflow execution?
+        delete_confirm: Are you sure that you want to delete this automated pipeline?
         edit_button: Edit
       create:
-        error: Could not create an automated workflow execution for %{workflow_name}
-        success: Automated workflow execution for %{workflow_name} was successfully created
+        error: Could not create an automated pipeline for %{workflow_name}
+        success: Automated pipeline for %{workflow_name} was successfully created
       destroy:
-        error: Could not delete automated workflow execution for %{workflow_name}
-        success: Automated workflow execution for %{workflow_name} was successfully deleted
+        error: Could not delete automated pipeline for %{workflow_name}
+        success: Automated pipeline for %{workflow_name} was successfully deleted
       edit_dialog:
-        title: 'Editing Automated Workflow Execution: %{workflow_name}'
+        title: 'Editing Automated Pipeline: %{workflow_name}'
       index:
-        add_new_automated_workflow_execution: New automated workflow execution
-        subtitle: Setup automated workflow executions to launch when paired-end data is uploaded to the project
-        title: Automated Workflow Executions
+        add_new_automated_workflow_execution: New automated pipeline
+        subtitle: Setup automated pipelines to launch when paired-end data is uploaded to the project
+        title: Automated Pipelines
       new_automated_workflow_execution_modal:
         title: "%{workflow} parameters"
       pipeline_selection_modal:
-        title: Workflow Selection
+        title: Pipeline Selection
       table:
         empty:
-          description: No automated workflow executions have been configured for this project
-          title: No automated workflow executions
+          description: No automated pipelines have been configured for this project
+          title: No automated pipelines
         headers:
           actions: Actions
           created_at: Created
           id: ID
           name: Name
           updated_at: Updated
-          workflow_name: Workflow Name
-          workflow_version: Workflow Version
+          workflow_name: Pipeline Name
```

Comment:
**nitpick (blocking):** I think the following 2 translations can be reverted

Reply:
Updated in 51f582d